### PR TITLE
mixin, variable

### PR DESCRIPTION
**Pull request by huynh-dac-phu**

### CODE REVIEW

**Comment by huynh-dac-phu** [comment]
ready

File: style/_mixin--media.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+$res_320 : "only screen and (min-width: 320px)";
+$res_480 : "only screen and (min-width: 480px)";
+$res_768 : "only screen and (min-width: 768px)";
+$res_1024: "only screen and (min-width: 1024px)";
```
Comment (chien-pv) [inline style/_mixin--media.scss:4]: Tách khai báo biến ra file riêng nhé

**Comment by chien-pv** [comment]
commented

**Comment by huynh-dac-phu** [comment]
ready

**Pull request merged**